content/index.md

```diff
@@ -15,11 +15,12 @@ image:
 og:
   img: default
 summary: |
-  **We help companies create better web applications**
-  with resilient design systems & well-tested code.
-  As experts in digital product design and development,
-  we focus on performance, accessibility, and architecture --
-  for low maintenance costs over the long term.
+  **As experts in digital product design and development,** 
+  we focus on performance, accessibility, and architecture -- 
+  delivering resilient design systems and well-tested code
+  for low maintenance costs over the long term.  We’ll help 
+  clarify & prioritize your goals as we work towards them 
+  together. 
```

Comment:
@SondraE I think a simple statement is good here since this is already a little jargony, but is there a good way to emphasize the "lower maintenance costs **for you**" benefit to the potential client ... something like: 

"As experts in digital product design and development, we focus on performance, accessibility, and architecture -- 
delivering resilient design systems and well-tested code resulting in lower maintenance costs for you.  We’ll help 
clarify & prioritize your goals as we work towards them together."

Reply:
**As experts in digital product design and development,** 
  we focus on accessibility, performance, and architecture. 
  We deliver resilient design systems and well-tested code
  giving you low maintenance costs for years.  We’ll help 
  clarify & prioritize your goals as we work toward them 
  together. 

How about this edit, @stacyk ?

Reply:
@SondraE love it